pam/internal/adapter/nativemodel.go

```diff
@@ -90,6 +90,7 @@ func (m *nativeModel) Init() tea.Cmd {
 
 	m.interactive = isSSHSession(m.pamMTx) || IsTerminalTTY(m.pamMTx)
 	rendersQrCode := m.isQrcodeRenderingSupported()
+	hasQrCode := m.serviceName != polkitServiceName
```

Comment:
```suggestion
	supportsQRCode := m.serviceName != polkitServiceName
```

Reply:
Indeed, but again I think that's for another reafactor PR... I definitely think that we should change the qr-code story as something else (not even device, since it is just a web-based authentication that can be done via another device or in the very same machine once we've a web-UI rendering it).

I had commented it already internally, I'll ping you about it.

Reply:
Sorry, I was referring to the second part of the sentence: about switching to `supportsDeviceAuth` for all

Reply:
Ok, then I think we're good to go here once you rename the variable introduced here.

Reply:
To avoid further confusion, my proposal is that we rename `hasQrCode` to `supportsDeviceAuth` and, possibly in a follow-up PR, rename `rendersQrCode` to `supportsQRCode`.

Reply:
Done.

Reply:
Oh, well I had followed your initial suggestion on the diff... However, I'd prefer to keep it mentioning qrcode, since that's what we do set depending on it.

We agree that `layouts.QrCode` is actually a bad name, but until we don't rename it I'd keep it "consistently" wrong in calling device-auth qrcode, or we end up adding too many cases here.

Reply:
Ok, fair enough